dataprofiler/profilers/order_column_profile.py

```diff
@@ -466,7 +467,7 @@ def _update_helper(self, df_series_clean: Series, profile: dict) -> None:
         Update col profile properties with clean dataset and its known null parameters.
 
         :param df_series_clean: df series with nulls removed
-        :type df_series_clean: pandas.core.series.Series
+        :type df_series_clean: polars.series.series.Series
```

Comment:
I put polars.series.series.Series since when you get the type of any polars series it returns <class 'polars.series.series.Series'>. However if you think I should change it to polars.Series still happy to do so.